### PR TITLE
Normal form

### PR DESCRIPTION
**Pull request by ederc**
Adds a first version of normal form computation

### CODE REVIEW

File: src/Rings/groebner.jl [hunk 1]
```diff
@@ -311,3 +311,121 @@ function leading_ideal(I::MPolyIdeal, ord::Symbol)
   return leading_ideal(groebner_basis(I, ord), ord)
 end
 
+@doc Markdown.doc"""
+    normal_form_internal(I::Singular.sideal, J::MPolyIdeal)
+
+**Note**: Internal function, subject to change, do not use.
+
+Compute the normal form of the generators of the ideal `I` w.r.t. a
```
Comment (thofma) [inline src/Rings/groebner.jl]: Which generators? `gens(I)`? I think we should try to spell it out.

Comment (ederc) [reply src/Rings/groebner.jl]: Alright, added `gens(I)` to the docu description.

**Comment by thofma** [comment]
Can you recreate the docstrings using 1.5? CI is doing the doctests with 1.5 and there was an annoying breaking change in the printing.

**Comment by fieker** [comment]
On Sat, May 22, 2021 at 01:12:08PM -0700, ederc wrote:
> @ederc commented on this pull request.
What about explaning what a normal form actually is?

**Pull request merged**